quantum/quantum.c

```diff
@@ -108,6 +108,10 @@ __attribute__((weak)) bool process_action_kb(keyrecord_t *record) {
 }
 
 __attribute__((weak)) bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    return process_record_json(keycode, record);
```

Comment:
Really not a fan of introducing this convention, its already bad enough that keyboards forget to call `process_record_user`. Now there will be a new convention, and a bunch of "broken" keyboards as they do `process_record_kb -> process_record_user`.

Reply:
I'm not married to this implementation, but other approaches seemed more "hacky".
Although keyboard's `process_record_kb` can certainly brake this, keyboards breaking convention might be an other problem we have to tackle in some other way.

I marked this PR to be Draft until we decide how to proceed.

Reply:
If anything, I would say that it should be moved into to the rest of the process_record_* handling is done, just like a normal feature.